### ISSUE TITLE
[Ente Auth] Full iOS device recovery will render auth unusable

### ISSUE THREAD

**Issue by erinacio** [ente/ente#3218]
### Description

- Did a full device backup of the previous iPhone.
- Restore backup on a new iPhone.
- Open Auth on the new phone, it only displays a looping circle with hamburger icon and search bar. Nowhere to find my codes, also can't add new codes.

Note that "export codes" also doesn't work, only results in an endless looping circle.

Screenshot of the main app screen attached:

![img_v3_02ej_f47f4c4b-8cfc-4036-b801-60fba8975chu](https://github.com/user-attachments/assets/277f6417-a174-4f2f-8215-cf6e044444d5)

### Version

v3.1.3

### What product are you using?

Ente Auth

### What platform are you using?

Mobile - iOS

**Comment by prateekmedia** [comment]
If the codes were synced i.e. You were usign auth logged in then simple logout/login would work or as suggested by @stevenwalton a reinstall would work.

**Comment by prateekmedia** [comment]
@stevenwalton Yes the offline codes won't get restored that way cause they were not synced. I think during this recovery the security key changes in iOS so the db won't get unlocked with same key. Any thoughts @ua741?

**Comment by AmanRajSinghMourya** [comment]
Also reproduced in #7282 when importing an Aegis backup after migrating to a new iPhone.
> Yes the offline codes won't get restored that way cause they were not synced. I think during this recovery the security key changes in iOS so the db won't get unlocked with same key.

Closing this as duplicate of https://github.com/ente/ente/issues/7282 and keeping https://github.com/ente/ente/issues/7282 open for tracking
